src/main/scala/org/constellation/API.scala

```diff
@@ -502,8 +502,13 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
         } ~
         pathPrefix("genesis") {
           path("create") {
-            entity(as[Set[Id]]) { ids =>
-              complete(Genesis.createGenesisAndInitialDistribution(dao.selfAddressStr, ids, dao.keyPair))
+            entity(as[Seq[AccountBalance]]) { balances =>
+              val go = Genesis.createGenesisAndInitialDistribution(
```

Comment:
This endpoint is only for old processes, right? It's not used anymore on the production use-case?

Reply:
I have searched the code-base and it is used only in `Simulation` but we need to ask @buckysballs probably if someone uses it 🤔